src/Flex.php

```diff
@@ -483,7 +483,9 @@ public function install(Event $event = null)
         $this->io->writeError('');
 
         if (!$recipes) {
-            $this->synchronizePackageJson($rootDir);
+            if (ScriptEvents::POST_UPDATE_CMD === $event->getName()) {
```

Comment:
This is borked, $event can be null when called via https://github.com/symfony/flex/blob/ff85b54092f70c866709cd97f0f1ffa1ddca2b24/src/Flex.php#L376

I'm not sure how to best fix this sorry so will leave that to you :)

Reply:
fixed in 4961515